testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/opentracing/OnOffOpenTracingTestCase.java

```diff
@@ -59,6 +60,7 @@
  *
  * @author Jan Stourac <jstourac@redhat.com>
  */
+@Ignore
```

Comment:
Emmanuel's PR includes a JIRA number so if you add that you will avoid conflicts.

Reply:
thanks, I've amended the commit in the #13756 branch too.